modules/data/db_manager.py

```diff
@@ -142,45 +142,51 @@ def count_from(table_name: str, select: str = "*", where: str = "", where_args:
                 try:
                     cur.execute(f"SELECT COUNT({select}) as number FROM {table_name} WHERE {where}", where_args)
                 except sqlite3.Error as e:
-                    logger.error(str(e))
+                    logger.error("DbManager.count_from: %s", e)
             else:
                 try:
                     cur.execute(f"SELECT COUNT({select}) as number FROM {table_name} WHERE {where}")
                 except sqlite3.Error as e:
-                    logger.error(str(e))
+                    logger.error("DbManager.count_from: %s", e)
         else:
             try:
                 cur.execute(f"SELECT COUNT({select}) as number FROM {table_name}")
             except sqlite3.Error as e:
-                logger.error(str(e))
+                logger.error("DbManager.count_from: %s", e)
```

Comment:
a bit redundant code here, you could create a function for this try except and make only try_err("different string") or similar